### PR TITLE
Fixes Issues

### PR DESCRIPTION
**Pull request by AMcilraith**

### CODE REVIEW

File: maps/torch/structures/signs.dm [hunk 1]
```diff
@@ -62,7 +62,7 @@
 	icon = 'maps/torch/icons/obj/solgov-decals.dmi'
 	icon_state = "solgovseal"
 
-/obj/structure/sign/ntefcrest
+/obj/structure/sign/solgovseal
 	name = "\improper NTEF Crest"
```
Comment (Codingale) [inline maps/torch/structures/signs.dm]: missed this: NTEF Crest -> Sol Gov Crest?

File: maps/torch/torch1_deck5.dmm [hunk 2]
```diff
@@ -1344,8 +1344,8 @@
 "cA" = (
 /obj/effect/overmap/visitable/ship/landable/guppy{
 	desc = "An SSE-U3 utility pod, broadcasting SC codes and the callsign \"Andromeda-3 Garuda\".";
-	name = "NTRP Garuda";
-	shuttle = "NTRP Garuda"
+	name = "G-U-P";
```
Comment (Codingale) [inline maps/torch/torch1_deck5.dmm:1347]: rip Garuda, is this the only few instances?

**Review by Codingale — APPROVED** [review]
Looks great to me, just the one change needed if you could do that.

File: maps/torch/torch1_deck5.dmm (hunk 2 above)
Comment (AMcilraith) [reply maps/torch/torch1_deck5.dmm:1347]: Yeah

File: maps/torch/structures/signs.dm (hunk 1 above)
Comment (AMcilraith) [reply maps/torch/structures/signs.dm]: Yes!

**Pull request merged**